File: ncDados/dSexo.vb
Namespace nsDados

  Public Class ColecaoSexo
    Inherits List(Of dSexo)
  End Class

  Public Class dSexo

    Private _codigo As String
    Private _descricao As String

    Public Property codigo() As String
      Get
        Return _codigo
      End Get
      Set(ByVal value As String)
        _codigo = value
      End Set
    End Property

    Public Property descricao() As String
      Get
        Return _descricao
      End Get
      Set(ByVal value As String)
        _descricao = value
      End Set
    End Property

  End Class

End Namespace

File: Modelos/dSexo.cs
using System;
using System.Collections.Generic;

namespace Modelos
{
    public class ColecaoSexo : List<dSexo>
    {
    }

    public class dSexo
    {
        private string _codigo;
        private string _descricao;

        public string codigo
        {
            get => _codigo;
            set => _codigo = value;
        }

        public string descricao
        {
            get => _descricao;
            set => _descricao = value;
        }
    }

}
